builder/build_test.go

```diff
@@ -472,3 +473,61 @@ func Test_deDuplicate(t *testing.T) {
 		})
 	}
 }
+
+func Test_pathInScope(t *testing.T) {
+	root, err := filepath.Abs(".")
+	if err != nil {
+		t.Fatalf("unexpected error during test setup: %s", err)
+	}
+
+	cases := []struct {
+		name         string
+		path         string
+		expectedPath string
+		err          bool
+	}{
+		{
+			name:         "can copy folders without any relative path prefix",
+			path:         "common/models/prebuilt",
+			expectedPath: filepath.Join(root, "common/models/prebuilt"),
+		},
+		{
+			name:         "can copy folders with relative path prefix",
+			path:         "./common/data/cleaned",
+			expectedPath: filepath.Join(root, "./common/data/cleaned"),
+		},
+		{
+			name: "error if relative path moves out of the current directory",
+			path: "../private",
+			err:  true,
+		},
+		{
+			name: "error if absolute path is outside of the current directory",
+			path: "/private/common",
+			err:  true,
+		},
+		{
+			name: "error if relative path moves out of the current directory, even when hidden in the middle of the path",
+			path: "./common/../../private",
+			err:  true,
+		},
```

Comment:
Based on my comment above, I think adding the following might catch that case (_I'm trying to use `suggestion` feature but I'm not sure if it works for multiple lines_)
```suggestion
		},
		{
			name: "error if path is the current directory",
			path: root,
			err:  true,
		},
```

Reply:
PTAL

Reply:
@LucasRoesler PTAL